python_modules/libraries/dagster-looker/dagster_looker/dagster_looker_translator.py

```diff
@@ -343,6 +351,30 @@ def get_group_name(
         Returns:
             Optional[str]: A Dagster group name for the LookML structure.
         """
+        _, lookml_structure_type, lookml_structure_props = lookml_structure
+
+        if lookml_structure_type == "explore":
+            lookml_explore_tags: Sequence[str] = lookml_structure_props.get("tags", [])
+
+            dagster_group_names = []
+            for lookml_explore_tag in lookml_explore_tags:
+                if not lookml_explore_tag.startswith(DAGSTER_LOOKER_GROUP_TAG_PREFIX):
+                    continue
+
+                _, _, dagster_group_name_from_lookml_tag = lookml_explore_tag.partition(
+                    DAGSTER_LOOKER_GROUP_TAG_PREFIX
+                )
+                dagster_group_name_from_lookml_tag = (
+                    dagster_group_name_from_lookml_tag.strip().replace(" ", "_")
+                )
+
+                dagster_group_names.append(dagster_group_name_from_lookml_tag)
+
+            if not dagster_group_names:
+                return None
+
+            return dagster_group_names.pop()
```

Comment:
If there are more than one it feels a little weird to just pick the first one. I wonder if we should ignore them in that case and warn?